assets/js/base/components/payment-methods/payment-method-options.js

```diff
@@ -8,17 +8,17 @@ import {
 	useEmitResponse,
 } from '@woocommerce/base-hooks';
 import { cloneElement } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
 import {
 	useEditorContext,
 	usePaymentMethodDataContext,
 } from '@woocommerce/base-context';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
-import Tabs from '../tabs';
 import PaymentMethodTab from './payment-method-tab';
```

Comment:
Should we change the name of this component from `PaymentMethodTab`?

Reply:
Because we no longer have tabs? Yes, that makes sense. Change to `PaymentMethodCard`.